### DISCUSSION TITLE
Does RTK 1.9.7 is compatible with chrome 49?

### DISCUSSION THREAD

**Discussion by mrdulin** [reduxjs/redux-toolkit#4123] [Q&A] [answered]
I'm maintaining a legacy project and preparing to migrate from redux to RTK. 

My web SPA is running in CEF, which uses Chrome 49.

![3616c72fedd864c53429389e6a6d9cae](https://github.com/reduxjs/redux-toolkit/assets/17866683/218a2dbc-73e6-43fd-8a3f-f58d083d3e30)

**Comment by phryneas** [comment]
If you are targeting a legacy browser, you will probably add transpilation for your libraries as well as some polyfills to your stack. At that point you could also use RTK 2.

The only feature that we use that cannot be transpiled away/polyfilled are Proxies, but those are already supported since Chrome 49.

**Reply by mrdulin** [reply]
Thanks for your reply. So without transpiling the RTK library in my project by Webpack and Babel, or adding any polyfills, is it compatible with Chrome 49? I'm afraid a certain usage will break my application.

**Reply by phryneas** [reply, marked as answer]
I have no idea.
Chrome 49 is an 8 year old browser that was already outdated when RTK entered alpha five years ago and we definitely are not *trying* to support that target, but RTK 1 did support IE, so maybe it will.

That said, with a target like that you should really use modern dependencies and transpile them - otherwise you will end up with a completely weird stack of unsupported legacy versions of dependencies (we won't ship any updates for RTK 1 anymore!).

**Reply by jorisw** [reply]
I was able to use Redux ToolKit in Chrome 47 by downgrading Immer to 9.x (via `resolutions` in `package.json` and then setting `useSetProxies(false)` on it:

```
import { enableES5, setUseProxies } from 'immer'
setUseProxies(false)
enableES5() // May need this too
```
